helm-charts/move2kube/templates/move2kube-api-stateful-set.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       labels:
         move2kube.konveyor.io/service: "{{ .Release.Name }}-move2kubeapi"
     spec:
-      {{- if .Values.persistentVolumeClaim.enable }}
+      {{- if and (.Values.deployment.api.initContainer) (.Values.persistentVolumeClaim.enable) }}
       initContainers:
         - name: change-perms
```

Comment:
Why is the init container used here?

Reply:
it's optional now, disabled by default. It's still useful to have on K8s clusters and more lenient Openshift clusters.